keyboards/aeboards/ext65/rev2/config.h

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2015 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4145 // "AE"
+#define PRODUCT_ID      0xA652 // AEboards EXT65 Rev2
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AEboards
+#define PRODUCT         AEboards Ext65 Rev2
+#define DESCRIPTION     AEboards Ext65 Rev2
```

Comment:
```suggestion
#define PRODUCT         Ext65 Rev2
```

Reply:
Not sure what the problem is here considering Rev1 and Aegis have this as well?

Reply:
Well, those should not be duplicating the `MANUFACTURER` string either. We have a new set of guidelines for PRs in the works: https://github.com/qmk/qmk_firmware/pull/9913

Reply:
@fauxpark are we going to debate this yet again?

https://github.com/qmk/qmk_firmware/pull/9024

Stop forcing people to adapt to the minority use case. It's their product, they should be allowed to name it as they see fit.

Reply:
I think this discussion would be better in an issue, and not in a PR.

But my 2 cents: major manufactures appear to be using the same configuration that we are encouraging... So, it would be nice to keep things consistent.